hasher-matcher-actioner/webapp/src/Settings.jsx

```diff
@@ -3,11 +3,189 @@
  */
 
 import React from 'react';
+import Card from 'react-bootstrap/Card';
+import Col from 'react-bootstrap/Col';
+import Row from 'react-bootstrap/Row';
+import Tabs from 'react-bootstrap/Tabs';
+import Tab from 'react-bootstrap/Tab';
+import ListGroup from 'react-bootstrap/ListGroup';
 
 export default function Settings() {
   return (
     <>
-      <h1>Settings</h1>
+      <Tabs defaultActiveKey="signals" id="setting-tabs">
+        <Tab eventKey="signals" title="Signals">
+          <SignalSettingsTab />
+        </Tab>
+        <Tab eventKey="pipeline" title="Pipeline">
+          Todo!
+        </Tab>
+      </Tabs>
+    </>
+  );
+}
+
+function SignalSettingsTab() {
+  return (
+    <>
+      <DataSets />
+      <ExternalSignalSources />
+    </>
+  );
+}
+
+function DataSets() {
+  return (
+    <>
+      <h2 className="mt-4">Datasets</h2>
+      <Row className="mt-3">
+        <DataSet />
+        <Col lg={4} sm={6} xs={12} className="mb-4">
+          <Card className="text-center">
+            <Card.Header className="text-white bg-success">
+              <h4 className="mb-0">Verified Cat Imagery</h4>
+            </Card.Header>
+            <Card.Subtitle className="mt-0 mb-0 text-muted">
+              Shared
+            </Card.Subtitle>
+            <Card.Body className="text-left">
+              <Card.Text>
+                <h5>Labels</h5>
+                <ul>
+                  <li>cat</li>
+                  <li>delete</li>
+                </ul>
+                <h5>Matching</h5>
+                <ul>
+                  <li>PDQ: Distance 31</li>
+                </ul>
+              </Card.Text>
+            </Card.Body>
+            <ListGroup variant="flush" className="text-left">
+              <ListGroup.Item>
+                <h5>Share To: Cat Dataset</h5>
+                <h6>Labels</h6>
+                <ul>
+                  <li>cat</li>
+                </ul>
+              </ListGroup.Item>
+            </ListGroup>
+            <a href="#todo" className="btn btn-primary">
```

Comment:
Maybe use `<Button>` here. Can still be rendered as an anchor tag if that's necessary / desired using the as attribute: `<Button as="a">Edit</Button>`